norm_flow_pytorch.py

```diff
@@ -298,6 +267,7 @@ def __init__(
         ):
         super(NormalisingFlow, self).__init__()
         self.size_in = size_in
+        self._gaussian_normalization = log(sqrt(pow(2*pi, size_in)))
```

Comment:
probably makes little difference but we don't need to calculate this everytime so I made the log_probability_norm a function of the class so we can cache it